README.md

```diff
@@ -118,7 +118,18 @@ spec:
 
 #### TLS
 
-TLS is optional but recommended for production deployments. It is used if `TLS_KEY` and `TLS_CERT` environment variables are set.
+TLS is optional but recommended for production deployments. It is used if `TLS_KEY` and `TLS_CERT` environment variables are set. Example:
+
+```bash
+docker run -p 8080:8080 -e TLS_KEY="$(cat sourcegraph.key)" -e TLS_CERT="$(cat sourcegraph.crt)" sourcegraph/lang-typescript
+```
+If you are using a self signed certificate [like this](https://docs.sourcegraph.com/admin/nginx#tls-https) add [-e NODE_TLS_REJECT_UNAUTHORIZED=0](https://github.com/nodejs/node/issues/5258) to the command line above.
+
+Make sure to use Websocket with SSL in your Sourcegraph settings to connect to the server:
+
+```json
+"typescript.serverUrl": "wss://localhost:8080"
```

Comment:
This URL is also only applicable to Docker

Reply:
Moved this under the Docker section